nilearn/glm/first_level/first_level.py

```diff
@@ -74,7 +78,7 @@ def mean_scaling(Y, axis=0):
 
 
 def _ar_model_fit(X, val, Y):
-    """Wrapper for fit method of ARModel to allow joblib parallelization"""
+    """Wrap for fit method of ARModel to allow joblib parallelization."""
```

Comment:
```suggestion
    """Wrapper for fit method of ARModel to allow joblib parallelization."""
```

Reply:
well flake8 will complain that this is not in the imperative mood, so maybe "create a wrapper" which is not ideal either.

suggestions are welcome

or we could silence that flake8 rule

Reply:
Ok maybe just "Wrap fit method..."